keyboards/akruvia/una/una.c

```diff
@@ -0,0 +1,23 @@
+// Copyright 2023 akruvia (@akruvia)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "una.h"
+
+void keyboard_pre_init_kb(void) {
+    setPinInputHigh(GP3);
+    setPinInputHigh(GP2);
+    keyboard_pre_init_user();
+}
+
+void matrix_scan_kb(void) {
+    if (!readPin(GP3)) {
+        register_code(KC_MS_BTN1);
+    } else {
+        unregister_code(KC_MS_BTN1);
+    }
+    if (!readPin(GP2)) {
+        register_code(KC_MS_BTN2);
+    } else {
+        unregister_code(KC_MS_BTN2);
+    }
```

Comment:
```suggestion
    }
    matrix_scan_user();
```

However hard coded logic at the keyboard level is fairly poor. The usual recommendation would be to implement a custom matrix and set additional matrix locations.

Reply:
For reference, the work_louder/micro has a couple of additional locations added to the matrix, like this.  It may be a good reference for this.